plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -153,7 +155,11 @@ static void installAppEngineRuntime(IFacetedProject project, IProgressMonitor mo
   }
 
   private static String findCloudSdk() {
-    return PathResolver.INSTANCE.getCloudSdkPath().toString();
+    File location = CloudSdkProvider.getDefaultSdkLocation();
+    if (location == null) {
+      throw new NullPointerException("Could not find Google Cloud SDK");
```

Comment:
It is called from the method immediately above which uses this NPE approach for communicating errors.  The text shows up in an error dialog.

But that's an excellent point re: prompting the user.